roles/router/templates/sriov-device-plugin-config.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriovdp-config
+data:
+  config.json: |
+    {
+      "resourceList": [
+        {
+          "resourcePrefix": "intel.com",
+          "resourceName": "intel_sriov_vfio_access",
```

Comment:
Is there any benefit for having 2 sets of resources (`*_vfio_access` and `*_vfio_core`)? Why not having a single set (e.g., `*_vfio`)

Reply:
In later stage, when we want to deploy multiple UPF's OR to use different PF's for N3/N6 then it will be helpful in that case. also it will be clear for the user which one will be used for N3/N6. So it would be better to use as above.